features/enqueuing_products_from_route.feature

```diff
@@ -0,0 +1,36 @@
+@enqueuing_products_from_route
+Feature: Browsing products items
+  In order to import the product from Akeneo
+  As an Administrator
+  I want to enqueue the product in Akeneo items queue
+
+  Background:
+    Given I am logged in as an administrator
+    And there is a product item with identifier "braided-hat-m"
+    And there is a product item with identifier "t-shirt-xl"
+    And there is a product item with identifier "braided-hat-l"
+
+  @ui
+  Scenario: Browsing all items
+    When I browse product item
+    Then I should see 3 products in the list
+
+  @ui
+  Scenario: Enqueue two products
+    When I browse product item
+    And I click "Schedule Akeneo PIM import" button on "braided-hat-m" product
```

Comment:
There should not be any mention to the used interface in the step description.
So this should be:
```
And I schedule an Akeneo PIM import for the "braided-hat-m" product
```